package.json

```diff
@@ -94,17 +94,5 @@
     "node": ">=10.18"
   },
   "license": "MIT",
-  "main": "public/main.js",
-  "window": {
-    "title": "Ungit",
-    "icon": "icon.png",
-    "toolbar": false,
-    "frame": false,
-    "show": false,
-    "width": 1000,
-    "height": 600,
-    "position": "center",
-    "min_width": 400,
-    "min_height": 200
-  }
+  "main": "public/main.js"
```

Comment:
Is `main` used by electron?

If not, as this is the [entrypoint when doing `require('ungit')`](https://docs.npmjs.com/files/package.json#main), does it make sense to point to a UI script?

Reply:
Yes, it is used by electron: https://www.electronjs.org/docs/tutorial/first-app#writing-your-first-electron-app